src/layouts/default.vue

```diff
@@ -1,12 +1,9 @@
 <template>
-  <div class="app grid h-screen overflow-hidden">
+  <div class="app grid h-screen">
```

Comment:
This breaks the separate scrolling of the filters sidebar and the main page. Currently we have this outer `div` full height, and its children have `overflow-y` set to `scroll`.

Reply:
I had a feeling there was a purpose to this but I didn't remember the filter sidebar, duh! I think we still can do it with this classes then.